app/tasks.py

```diff
@@ -0,0 +1,15 @@
+from sqlalchemy.orm import Session
+
+from app import crud
+from app.schemas import LocationCreate, PriceBase
+
+
+def create_price_location(db: Session, price: PriceBase):
```

Comment:
are `Session` objects thread-safe? As I assume we're using threads to launch a task in the background.

Reply:
No idea 😅
I used the background tasks example from the fastAPI tutorials..

Reply:
though they do recommend to create a new db session here: https://github.com/tiangolo/fastapi/discussions/8502#discussioncomment-5606921

Reply:
added it to #31